varya/assets/sass/base/_editor.scss

```diff
@@ -30,12 +30,19 @@ a {
 	cursor: pointer;
 }
 
-// Gutenberg injects a rule that limits the max width of .wp-block to 580px
-// This line overrides it to use the responsive spacing rules for default width content
 .wp-block {
+
+	// Gutenberg injects a rule that limits the max width of .wp-block to 580px
+	// This line overrides it to use the responsive spacing rules for default width content
 	&:not([data-align="full"]):not([data-align="wide"]){
 		max-width: var(--responsive--aligndefault-width);
 	}
+
+	// Use the theme's max-width for wide alignment.
+	&[data-align="wide"],
+	&.alignwide {
+		max-width: var(--responsive--alignwide-width);
```

Comment:
I opted to use this variable instead of `@extend %responsive-alignwide-width;` here because we do not need to apply `margin-left` and `margin-right` rules.

Reply:
Ahhh, clever! Happy to see it was easy to sort out… thought I might need to write another responsive logic system just for the editor 👍